sdcm/cluster.py

```diff
@@ -3771,10 +3771,11 @@ def get_monitoring_data_stack(self, node):
         node.remoter.send_files(src=sct_dashboard_file, dst=sct_monitoring_addons_dir)
 
         annotations_json = self.get_grafana_annotations(node)
-        tmp_dir = tempfile.mkdtemp()
-        with io.open(os.path.join(tmp_dir, 'annotations.json'), 'w', encoding='utf-8') as f:
-            f.write(annotations_json)
-        node.remoter.send_files(src=os.path.join(tmp_dir, 'annotations.json'), dst=sct_monitoring_addons_dir)
+        if annotations_json:
```

Comment:
add else with log warning that annotations not found

Reply:
There is already a warning in get_grafana_annotations()

```
logger.warning("unable to get grafana annotations [%s]", str(ex))
```